sklearn/metrics/tests/test_regression.py

```diff
@@ -86,6 +91,9 @@ def test_multioutput_regression():
     error = mean_absolute_error(y_true, y_pred)
     assert_almost_equal(error, (1. + 2. / 3) / 4.)
 
+    error = np.around(mean_absolute_percentage_error(y_true, y_pred),
+                      decimals=2)
+    assert_almost_equal(error, 375e+14, decimal=-14)
```

Comment:
Same remark as above for this assertion.

Reply:
done